bsdlib/doc/CHANGELOG.rst

```diff
@@ -5,6 +5,23 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+bsdlib 0.7.0
+************
+
+* Major rewrite of the lower transport layer to fix an issue where packages where lost in a high bandwidth application.
```

Comment:
small typo: 

> [...] fix an issue where packages **where** lost [...]

Reply:
This was corrected in my commit, must have been overwritten?
https://github.com/NordicPlayground/nrfxlib/commit/c50ae953ae19a61daca75a44fc4821ae9f325e83